test/libyul/yulOptimizerTests/fullSuite/stack_compressor_msize.yul

```diff
@@ -54,6 +54,7 @@
 //         sstore(not(gcd(_3, _2)), _1)
 //         sstore(2, _1)
 //         extcodecopy(_1, msize(), _1, _1)
+//         sstore(0, 0)
```

Comment:
I have **NO** idea why this was missing before - maybe there is a really bad bug lurking here somewhere...

Reply:
That's really a bit weird...

Reply:
Or wait... did "before" have the equal store eliminator merged :-)? If so, then that's the reason I guess.

Reply:
Yeah, I guess that's it. And the equal store eliminator removes the second copy, while the redundant store eliminator removes the first...

Reply:
But even the equal store eliminator cannot remove this, can it?
It might be that the test results were just wrong for an earlier commit...

Reply:
ok sure, but you cannot have both of them missing?

By the way, do we actually still need the equal store eliminator?

Reply:
The diff I'm seeing merely moves the ``sstore(0, 0)`` three lines down...

Reply:
The input contains two ``sstore(0, 0)``, the output contained one of them before the diff, and still contains one of them after the diff, it just removes the other one.

Reply:
But yeah, I didn't get the chance to look at the equal store eliminator, but it did sound redundant to me relative to the redundant store eliminator :-).